tests/sysctl/test_sysctl.py

```diff
@@ -175,15 +177,15 @@ def test_proto_get_tuple(self) -> None:
 
     def test_proto_get_unknown(self) -> None:
         self.proto.lineReceived(b'net.unknown')
-        self.assertEqual(b'[error] net.unknown not found\n', self.tr.value())
+        self.assertEqual(b'[error] path net.unknown not found\n', self.tr.value())
```

Comment:
I don't think we should change the error message in this PR.

Reply:
Ok